### DISCUSSION TITLE
How to check which client use upstream server?

### DISCUSSION THREAD

**Discussion by k-matti** [AdguardTeam/AdGuardHome#6985] [Q&A]
![image](https://github.com/AdguardTeam/AdGuardHome/assets/66017397/17aa3da2-4897-4d63-a223-3b0e20d57b9d)
Is it possible to find out which client call my router 3 times instead use AdGuard?

**Comment by vay1314** [comment]
I also want to know the answer to this question

**Reply by Dynamic5912** [reply]
It could be AGH doing PTR requests back to the router. 

But I'm not 100% sure.
